build.sbt

```diff
@@ -20,7 +23,11 @@ lazy val root = project.in(file("."))
     scalaVersion       :=  "2.11.12",
     crossScalaVersions :=  Seq("2.11.12", "2.12.5"),
     scalacOptions      :=  BuildSettings.compilerOptions,
-    javacOptions       :=  BuildSettings.javaCompilerOptions
+    javacOptions       :=  BuildSettings.javaCompilerOptions,
+
+    siteSubdirName in ScalaUnidoc := "latest/api",
+    addMappingsToSiteDir(mappings in (ScalaUnidoc, packageDoc), siteSubdirName in ScalaUnidoc),
+    gitRemoteRepo := "git@github.com:snowplow/scala-maxmind-iplookups.git"
```

Comment:
I think it'd be worth it to extract those settings in their own `docSettings` variable and store them in `BuildSettings`, also you can have the remote as https: `https://github.com/snowplow/scala-maxmind-iplookups.git`